core/src/main/java/hudson/BulkChange.java

```diff
@@ -25,6 +25,7 @@
 
 import hudson.model.Saveable;
 
+import java.io.Closeable;
```

Comment:
Well `Closeable extends AutoCloseable` so this works as well. I do not see any harm in implementing the stronger `Closeable`.